modules/ROOT/pages/sysconfig-enabling-wifi.adoc

```diff
@@ -0,0 +1,122 @@
+iwlwifi-dvm-firmware - DVM Firmware for Intel(R) Wireless WiFi adapters
+iwlwifi-mvm-firmware - MVM Firmware for Intel(R) Wireless WiFi adapters
+libertas-firmware - Firmware for Marvell Libertas SD/USB WiFi Network Adapters
+mt7xxx-firmware - Firmware for Mediatek 7600/7900 series WiFi/Bluetooth adapters
+nxpwireless-firmware - Firmware for NXP WiFi/Bluetooth/UWB adapters
+realtek-firmware - Firmware for Realtek WiFi/Bluetooth adapters
+tiwilink-firmware - Firmware for Texas Instruments WiFi/Bluetooth adapters
+atmel-firmware - Firmware for Atmel at76c50x wireless network chips
+zd1211-firmware - Firmware for wireless devices based on zd1211 chipset
+----
+
+For example, if a system has a Qualcomm wireless card then adding the `NetworkManager-wifi` and `atheros-firmware` packages would sufficiently enable the system for connecting to Wi-Fi.
+
+
+== When installing Fedora CoreOS
+
+For new systems the packages can be added using the xref:os-extensions.adoc[Adding OS Extensions] workflow. A NetworkManager configuration for the Wi-Fi connection will also need to be added so the system knows which wireless network to connect to. For more information on network configuration in Fedora CoreOS see xref:sysconfig-network-configuration.adoc[Network Configuration].
+
+An example Butane config that combines the extension and network configuration is shown below.
+
+.Butane config for Wi-Fi enablement
+[source,yaml,subs="attributes"]
+----
+variant: fcos
+version: {butane-latest-stable-spec}
+systemd:
+  units:
+    # Enable Wi-Fi in NetworkManager for an Intel wireless card
+    - name: rpm-ostree-install-wifi.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Enable Wi-Fi
+        Wants=network-online.target
+        After=network-online.target
+        Before=zincati.service
+        ConditionPathExists=!/var/lib/%N.stamp
+        [Service]
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive NetworkManager-wifi iwlwifi-dvm-firmware
+        ExecStart=/bin/touch /var/lib/%N.stamp
+        [Install]
+        WantedBy=multi-user.target
+storage:
+  files:
+    - path: /etc/NetworkManager/system-connections/wifi-guest.nmconnection
+      mode: 0600
+      contents:
+        inline: |
+          [connection]
+          id=wifi-guest
+          type=wifi
+          autoconnect=true
+          [wifi]
+          cloned-mac-address=permanent
+          mode=infrastructure
+          ssid=guest
+          mac-address=ab:cd:01:02:03:04
+          [wifi-security]
+          auth-alg=open
+          key-mgmt=wpa-psk
+          psk=PASSWORD
+          [ipv4]
+          method=auto
+----
+
+NOTE: When installing a system and adding WiFi enablement in this way the system will need to be on a wired network for the initial install since it will need to use the network to retrieve the WiFi enabling packages.
```

Comment:
For consistency

```suggestion
NOTE: When installing a system and adding Wi-Fi enablement in this way the system will need to be on a wired network for the initial install since it will need to use the network to retrieve the Wi-Fi enabling packages.
```